tests/integration/pages/experiment_timeline_and_population.py

```diff
@@ -70,24 +70,21 @@ def population_precentage(self):
     @population_precentage.setter
     def population_precentage(self, text=None):
         element = self.find_element(*self._population_precentage_locator)
+        element.clear()
```

Comment:
so vs code did some formatting... 
adding comment to help point out the line added here